src/index.tsx

```diff
@@ -1,13 +1,25 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import { HashRouter as Router, Routes, Route } from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { HomePage } from './pages/HomePage';
+import { PeoplePage } from './pages/PeoplePage/PeoplePage';
+import { NotFoundPage } from './pages/NotFoundPage';
 
 createRoot(document.getElementById('root') as HTMLDivElement).render(
   <Router>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route index element={<HomePage />} />
+        <Route path="people" element={<PeoplePage />}>
+          <Route path=":personId?" element={<PeoplePage />} />
+        </Route>
+        <Route path="*" element={<NotFoundPage />} />
+      </Route>
+    </Routes>
     <App />
```

Comment:
The <App /> component is rendered twice: once within the <Routes> and once outside. This redundancy might cause unexpected behavior. Consider removing the second <App /> rendering outside the <Routes>.